lib/tasks/sanitize_data.rake

```diff
@@ -0,0 +1,162 @@
+require 'highline'
+
+namespace :ofn do
+  namespace :data do
+    desc 'Sanitize data'
+    task sanitize: :environment do
+      guard_and_warn
+
+      # Sanitize base entities
+      canned_users = build_canned_users
+      disable_user_confirmation_emails
+      sanitize_batch("Spree::User", :sanitize_user, canned_users)
+      sanitize_batch("Spree::Customer", :sanitize_customer, canned_users)
+      sanitize_batch("Spree::Order", :sanitize_email, canned_users)
+      sanitize_batch("Spree::TokenizedPermission", :sanitize_tokenized_permission)
+      sanitize_batch("Spree::Address", :sanitize_address, canned_users)
+
+      # Sanitize payments related entities
+      sanitize_batch("Spree::PaymentMethod", :sanitize_payment_method)
+      sanitize_batch("Spree::CreditCard", :sanitize_credit_card)
+      sanitize_batch("Spree::Payment", :sanitize_payment)
+      sanitize_batch("Spree::PaypalExpressCheckout", :sanitize_paypal_express_checkout)
+      sanitize_batch("StripeAccount", :sanitize_stripe_account)
+
+      # Can't find the model for spree_paypal_accounts table (email and payer_id need to be deleted)
+      #   sanitize_batch("Spree::PaypalAccount", :sanitize_paypal_account, canned_users)
```

Comment:
it might not be baked by an AR model. I couldn't find anything either

Reply:
:+1:

Reply:
I removed the comment